extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -361,9 +382,12 @@ class AmpYoutube extends AMP.BaseElement {
 
   /** @private */
   buildImagePlaceholder_() {
+    // TODO: add custom placeholder logic for live channels
```

Comment:
Basically we have a problem here. Channels don't have default placeholder images. Channels have thumbnails, which are more like profile pictures for the channel, which can be retrieved using the Youtube Data API. The channel will have a currently streaming video, which will have a placeholder image, but there doesn't seem to be a good way to get the currently streaming video's `videoId` from the channelId without accessing an API token either.

Reply:
We should just do nothing (ie. don't call this if `hasPlaceholder || isLiveStream`) and just document that unlike normal yt videos, live channels don't have a default placeholder and they should add their own. This is fine, most components don't have a default anyway ( e.g. amp-twitter)